src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -98,27 +118,61 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewIni
     return this._isSelected;
   }
 
+  /**
+   * Indicates whether users can change the order of the repeater item.
+   * The repeater component's reorderable property must also be set to `true`.
+   */
   @Input()
-  public selectable: boolean = false;
+  public reorderable: boolean = false;
 
+  /**
+   * Indicates whether to display a checkbox in the left of the repeater item.
+   */
   @Input()
-  public reorderable: boolean = false;
+  public selectable: boolean = false;
 
+  /**
+   * Indicates whether to display an inline form within the repeater.
+   * Users can toggle between displaying and hiding the inline form.
+   */
   @Input()
   public showInlineForm: boolean = false;
 
+  /**
+   * Specifies an object that the repeater component returns for this repeater item
+   * when the `orderChange` event fires. Required if you set the `reorderable` property to `true`.
```

Comment:
I'd update the second sentence to make it a complete sentence instead of a fragment: `This is required if you set...`

Reply:
Done.